server/rest/src/test/java/org/infinispan/rest/ServerRestTestBlockHoundIntegration.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.rest;
+
+import org.kohsuke.MetaInfServices;
+
+import com.arjuna.ats.internal.arjuna.coordinator.ReaperThread;
+import com.arjuna.ats.internal.arjuna.coordinator.ReaperWorkerThread;
+
+import reactor.blockhound.BlockHound;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+@MetaInfServices
+public class ServerRestTestBlockHoundIntegration implements BlockHoundIntegration {
+   @Override
+   public void applyTo(BlockHound.Builder builder) {
+      // Let arjuna block - sometimes its thread will be put in our non blocking thread group
```

Comment:
We use Arjuna in core tests as well, maybe we should put the exception in the core module instead?

Reply:
We may need to. My guess is the reason is because we start arjuna in the main thread so the thread group for its thread isn't marked as non blocking.